images/linux/scripts/installers/nodejs.sh

```diff
@@ -9,6 +9,16 @@ curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-inst
 ~/n/bin/n lts
 npm install -g grunt gulp n parcel-bundler typescript newman
 npm install -g --save-dev webpack webpack-cli
+
+# Install the Netlify CLI using --unsafe-perm=true --allow-root options to avoid permission issues
+sudo npm i --unsafe-perm=true --allow-root -g netlify-cli
```

Comment:
we don't need `sudo` here since the whole provisioner runs with `sudo` permissions

Reply:
Thank you, fixed